maven-java/README.adoc

```diff
@@ -0,0 +1,37 @@
+= Maven Example
```

Comment:
Hi @kameshsampath could you make the README be markdown instead (README.md)?, this would consistent with the format we use in all our READMEs across OpenWhisk